packages/flutter_tools/test/general.shard/runner/runner_test.dart

```diff
@@ -19,7 +20,10 @@ import '../../src/context.dart';
 
 void main() {
   group('runner', () {
+    BufferLogger bufferLogger;
```

Comment:
IIRC `testUsingContext` already injects a `BufferLogger` called `testLogger` or something like that.